### PR TITLE
account_invoice_proforma_today module

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

**Comment by OSguard** [comment]
@rami-wafaie is our employee and so covert to initOS CLA 

@rami-wafaie can you update this PR with the new OCA guidelines (short header, Version number ..)

**Comment by OSguard** [comment]
:+1: thanks

**Comment by OSguard** [comment]
@alexis-via can you make a review here?

**Comment by codingforfun** [comment]
:+1:

**Comment by OSguard** [comment]
@sebastienbeau please review and merge

File: account_invoice_proforma_today/README.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Account Invoice - Proforma Today
+================================
+
+* The module extends the account.invoice model to set the date of the invoice as the today's
+ date when changing state from 'proforma' to 'open'.
+
+Installation
+============
+
+* No thing is required to install this module
```
Comment (elicoidal) [inline account_invoice_proforma_today/README.rst]: nothing

File: account_invoice_proforma_today/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+{
+    'name': 'Account Invoice Proforma Today',
+    'version': '7.0.1.0.0',
+    'author': 'initOS GmbH, Odoo Community Association (OCA)',
+    'category': '',
+    'description':
+    """
+    The module extends the account.invoice model
+    to set the date of the invoice as the today's
```
Comment (elicoidal) [inline account_invoice_proforma_today/__openerp__.py]: s/as the today's/as today's

File: account_invoice_proforma_today/README.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================
+Account Invoice - Proforma Today
+================================
+
+* The module extends the account.invoice model to set the date of the invoice as the today's
```
Comment (elicoidal) [inline account_invoice_proforma_today/README.rst]: s/as the today's/as today's

**Comment by elicoidal** [comment]
minor changes. LGTM :+1:

**Comment by OSguard** [comment]
Now it can be merged

**Comment by OSguard** [comment]
@elicoidal can you merge it

**Pull request closed without merging**